mitiq/zne/inference.py

```diff
@@ -1637,6 +1639,139 @@ def reduce(self) -> float:
 ]
 
 
+class ExpBayesFactory(BatchedFactory):
+    """Factory object implementing a zero-noise extrapolation algorithm based on
+    Bayesian Inference. The exponential ansatz for the expecation value:
+        E(lambda) = a + b * e**(-c*lambda),
+    where a, b and c are model parameters that need to be estimated,
+    while lambda is the noise scale factor.
+
+    Args:
+        scale_factors: Sequence of noise scale factors at which
+                       expectation values should be measured.
+        shot_list: Optional sequence of integers corresponding to the number
```

Comment:
It seems like this is not used, maybe we should note that it is ignored?

Reply:
This is supposed to be handled in `BatchedFactory` and its methods. So I don't think a note is necessary here.

Reply:
I made a note below, but by using that in this class as well, you may be able to improve the inferential quality by eliminating the noise parameter.